src/ga/individual.rs

```diff
@@ -1,43 +1,41 @@
 use std::fmt::Debug;
-
 use serde::Serialize;
 
-pub trait Gene: Sized + Default + Copy + Debug {}
 
-// Blanket implementaion.
-impl<T: Sized + Default + Copy + Debug> Gene for T {}
+pub trait Chromosome: Sized + Sync + Send + Clone + Default + Debug {}
 
-pub type Chromosome<T> = Vec<T>;
+/// Blanket implementation
+impl<T: Sized + Sync + Send + Clone + Default + Debug> Chromosome for T {}
```

Comment:
This is a blanket trait implementation, so that any reasonable type works with our API.

As of now, I do not have any knowledge about impact of blanket trait implementation on crate's compilation time, but I propose deferring this conversation for later.